### DISCUSSION TITLE
v8 doesn't work on fresh install of Laravel 7.x

### DISCUSSION THREAD

**Discussion by mrpritchett** [spatie/laravel-medialibrary#1963] [Q&A]
Running `composer require "spatie/laravel-medialibrary:^8.0.0"` returns `Script @php artisan package:discover --ansi handling the post-autoload-dump event returned with error code 1` using MacOS Catalina 10.15.5 (19F101) with Laravel Valet latest, including PHP7.4

**Comment by freekmurze** [comment]
Both on my local machine and on the GitHub actions it works. We didn't get any other reports, so I'm assuming this is a problem unique to your environment.
